common_design_subtheme/common_design_subtheme.libraries.yml

```diff
@@ -3,6 +3,15 @@ global-styling:
     theme:
       css/styles.css: {}
 
+# Arabic font using Dubai.
+# @todo This of the sake of the demo and could be merged with the base theme one
+# or removed.
+fonts-arabic:
+  label: Arabic
+  css:
+    theme:
+      css/fonts-arabic.css: {}
+
```

Comment:
This libraries definition seems like it affects subtheme implementations. Perhaps we can leave it here commented out like some of the other examples.